topics/replication.md

```diff
@@ -321,3 +321,16 @@ It is not possible to partially resynchronize a replica that restarted via the
 AOF file. However the instance may be turned to RDB persistence before shutting
 down it, than can be restarted, and finally AOF can be enabled again.
 
+Maxmemory on replicas
+---
+
+Replicas don't honor `maxmemory` because by default a replica will ignore this setting (unless it is promoted to master after a failover or manually). It means that the eviction of keys will be just handled by the master, sending the DEL commands to the replica as keys evict in the master side.
```

Comment:
```suggestion
Replicas don't honor `maxmemory` because by default a replica will ignore this setting (unless it is promoted to master after a failover or manually). It means that the eviction of keys will just be handled by the master, sending the DEL commands to the replica as keys evict in the master side.
```